template/src/pages/RoutingErrorPage.tsx

```diff
@@ -0,0 +1,30 @@
+import { useNavigate } from 'react-router-dom';
+import ErrorState from '@rio-cloud/rio-uikit/ErrorState';
+
+import { routes } from '../routes/routes';
+
+const RoutingErrorPage = () => {
+    const navigate = useNavigate();
+    const emailAddress = `${import.meta.env.VITE_SUPPORT_ADDRESS}`;
+
+    return (
+        <ErrorState
+            headline={'TODO ADD YOUR LOCALIZED ERROR MESSAGE HEADLINE'}
```

Comment:
Here we can use the Phrase shared project of common messages where some common error messages are already contained. Just extend them there

Reply:
Good catch! I've updated the project (and fixed the mergeTranslations script)